### PR TITLE
Javadoc for the  *stream in stream package

### PR DESCRIPTION
**Pull request by JkSelf**
A subtask of #91
Complete the javadoc for the  *stream in stream package.

### CODE REVIEW

File: src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java [hunk 1]
```diff
@@ -67,28 +67,78 @@
...
+   * @throws IOException if an I/O error occurs.
+   */
   public CTRCryptoInputStream(Properties props, ReadableByteChannel in,
       byte[] key, byte[] iv)
       throws IOException {
     this(props, in, key, iv, 0);
   }
 
+  /**
+   *Constructs a {@link com.intel.chimera.stream.CTRCryptoInputStream}.
```
Comment (sundapeng) [inline src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java]: *`blank`Constructs

File: src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java [hunk 2]
```diff
@@ -67,28 +67,78 @@
...
+   * @param iv Initialization vector for the cipher.
+   * @throws IOException if an I/O error occurs.
+   */
   public CTRCryptoInputStream(InputStream in, Cipher cipher, int bufferSize,
       byte[] key, byte[] iv) throws IOException {
     this(in, cipher, bufferSize, key, iv, 0);
   }
 
+  /**
+   *Constructs a {@link com.intel.chimera.stream.CTRCryptoInputStream}.
```
Comment (sundapeng) [inline src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java]: *`blank`Constructs

File: src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java [hunk 3]
```diff
@@ -98,30 +148,85 @@ public CTRCryptoInputStream(
...
+   * @param streamOffset the start offset in the stream.
+   * @throws IOException if an I/O error occurs.
+   */
   public CTRCryptoInputStream(ReadableByteChannel in, Cipher cipher,
       int bufferSize, byte[] key, byte[] iv, long streamOffset) throws IOException {
     this(new ChannelInput(in), cipher, bufferSize, key, iv, streamOffset);
   }
 
+  /**
+   *Constructs a {@link com.intel.chimera.stream.CTRCryptoInputStream}.
```
Comment (sundapeng) [inline src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java]: *`blank`Constructs

File: src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java [hunk 4]
```diff
@@ -353,22 +498,42 @@ protected byte postDecryption(long position) throws IOException {
     return padding;
   }
 
+  /**
+   *Gets the counter for input stream position.
```
Comment (sundapeng) [inline src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java]: *`blank`Gets

File: src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java [hunk 5]
```diff
@@ -353,22 +498,42 @@ protected byte postDecryption(long position) throws IOException {
...
+   *
+   * @param position the given position in the data.
+   * @return the counter for input stream position.
+   */
   protected long getCounter(long position) {
     return position / cipher.getTransformation().getAlgorithmBlockSize();
   }
 
+  /**
+   *Gets the padding for input stream position.
```
Comment (sundapeng) [inline src/main/java/com/intel/chimera/stream/CTRCryptoInputStream.java]: *`blank`Gets

**Comment by sundapeng** [comment]
Thank @JkSelf , I left some minor comments.

**Comment by JkSelf** [comment]
Thanks @sundapeng comments. I have fixed.

**Comment by sundapeng** [comment]
Thank @JkSelf for your contribution

**Pull request merged**